### ISSUE TITLE
Dependency-Tree

### ISSUE THREAD

**Issue by xRate1337** [bgnetworks/meta-dependencytrack#7]
Does someone knows how I can create a SBOM of my Yocto Project, that shows the dependency tree in Dependency-Track of my packages?

**Comment by Jasper-Ben** [comment]
I do not think that is a simple feat. You could try experimenting with extracting the necessary information from the dependency dot file generated with `bitbake -g`, see https://docs.yoctoproject.org/bitbake/2.8/bitbake-user-manual/bitbake-user-manual-intro.html#the-bitbake-command
